src/06-positron-rag/lib/insert/prepareArticlesCollection.ts

```diff
@@ -110,7 +110,17 @@ export async function prepareArticlesCollection() {
         },
       },
       {
-        name: "head",
+        name: "sectionIndex",
+        dataType: ["int"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "section",
```

Comment:
`section` is the main new field here (the GH diff slices this up funny, but this is the relevant new config in the schema definition)